ts/components/LabelledItem.tsx

```diff
@@ -169,9 +173,24 @@ export class LabelledItem extends React.Component<Props, State> {
               onChangeText={this.handleOnChangeText}
               onFocus={this.handleOnFocus}
               onBlur={this.handleOnBlur}
+              testID={`${this.props.testID}Input`}
             />
           )}
         </Item>
+        {this.props.description && (
+          <Item style={styles.noBottomLine}>
+            <H5
+              weight={"Regular"}
+              color={
+                this.props.isValid === undefined || this.props.isValid === true
```

Comment:
```suggestion
                this.props.isValid === false ? "red" : "bluegreyDark"
```

Reply:
Solved in [b6566b2](https://github.com/pagopa/io-app/pull/2833/commits/b6566b26d03764a9b1ae8a9cac127715a9fbb72f)